.circleci/config.yml

```
@@ -48,6 +48,7 @@ jobs:
           at: .
       - run: yarn build
       - run: yarn typecheck
+      - run: for d in packages/*; do cd $d && yarn build; done
```

You usually use tools like [wsrun](https://www.npmjs.com/package/wsrun) instead of looping like this. In this case I'd simply use:
```suggestion
      - yarn workspace @trusttoken-smart-contracts/contracts-por build
```